cmd/notary/integration_test.go

```diff
@@ -183,6 +183,131 @@ func TestInitWithRootKey(t *testing.T) {
 	require.Error(t, err, "Init with wrong role should error")
 }
 
+func TestInitWithRootCert(t *testing.T) {
+	setUp(t)
+
+	// key pairs
+	privStr := `-----BEGIN EC PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
+DEK-Info: AES-256-CBC,c9ccb4ef1effa1a080030c9c36942e8e
+role: root
+
+3pCHAMGD2QJDr8BAojd01wa4nzhct0Brk6olIAoaL9yRfV5jRguidu1UaoA22Tan
+9zOatIkxIgqkEP+P3+prIipbXJPbr9I9zVdWxhANSEhmQ95jmlk9syi/xeJT2oXB
+6+u84t59l0mRpuAisdC9AGkw7Cz2T5U51lhyCWjLDqE=
+-----END EC PRIVATE KEY-----`
+
+	certStr := `-----BEGIN CERTIFICATE-----
+MIIBWDCB/6ADAgECAhBKKoVsRNJdGsGh6tPWnE4rMAoGCCqGSM49BAMCMBMxETAP
+BgNVBAMTCGRvY2tlci8qMB4XDTE3MDQyODIwMTczMFoXDTI3MDQyNjIwMTczMFow
+EzERMA8GA1UEAxMIZG9ja2VyLyowWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQQ
+6RhA8sX/kWedbPPFzNqOMI+AnWOQV+u0+FQfeNO+k/Uf0LBnKhHEPSwSBuuwLPon
+w+nR0YTdv3lFaM7x9nOUozUwMzAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYI
+KwYBBQUHAwMwDAYDVR0TAQH/BAIwADAKBggqhkjOPQQDAgNIADBFAiA+eHPInhLJ
+HgP8nha+UqdYgq8ZCOlhdGTJhSdHd4sCuQIhAPXqQeWhDLA3/Pf8B7G3ZwWpPbZ8
+adLwkjqoeEKMaAXf
+-----END CERTIFICATE-----`
+
+	nonMatchingKeyStr := `-----BEGIN EC PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
+DEK-Info: AES-256-CBC,fd6e6735232efbc1a851549d12b8203d
+role: root
+
+Z6u+cAZOEmeoieyQHt6Lp8ZmLWPiyGXT0wTkfYMnGxZ+EX+6sBeu9CWgx+3kOCWQ
+qXuLmBjJ4ZwL/lZejeLLefF7jILA0oDLJtNH1L0oP7H/i7DUtNv+7Jvnci986Rx0
+i85wnaTwOgWv8n6q3tavmnIA/v2QqsTpmI+bhwrPNKQ=
+-----END EC PRIVATE KEY-----`
+
+	//set up notary server
+	server := setupServer()
+	defer server.Close()
+
+	//set up temp dir
+	tempDir := tempDirWithConfig(t, `{
+		"trust_pinning" : {
+			"disable_tufu" : false
```

Comment:
typo: `"disable_tofu"` - this shouldn't affect behavior since `disable_tofu` will disable to `false` anyhow, but would be nice to be explicit here.